packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -336,6 +336,159 @@ describe('Checkout', () => {
             });
         });
 
+        describe('invalidFieldsSummary ::', () => {
```

Comment:
Maybe not for this PR but i think we should consider using `describes` a little more, it would help structure the test and make it easier to follow. Example:

``` javascript
describe('invalidFieldsSummary ::', () => {...
  describe('when invalidFieldCount is falsey', () => {
     it('should return null')
  })
describe('when invalidFieldCount is truthy', () => {
     describe('and error state is equal to 1')
       it('should match the single error snapshot')
     describe('and error state is not equal to 1')
       it('should match the multi error snapshot')
  })
```

Reply:
ah yeah thank you, will definitely keep this in mind for the future - We've been using more `describes` in our  f-header component tests for checking all the different tenants and it's been really helpful as an extra separation between tests - super useful for error handling 😄